src/ensembl/src/shared/components/instant-download/instant-download-fetch/fetchSequenceChecksums.ts

```diff
@@ -59,10 +74,37 @@ type Variables = {
   transcriptId: string;
 };
 
-export const fetchTranscriptChecksums = (variables: Variables) =>
-  client
-    .query<GeneFragment>({
+export const fetchTranscriptSequenceData = (
```

Comment:
I am not sure about a better name but this sounds like we are fetching the sequence within this function itself.

Reply:
We are fetching information about the sequence that will later be used to fetch the sequence itself. Only it's no longer just checksums.

Would fetchTranscriptSequenceMetadata be better? 😄

Reply:
Yes, `fetchTranscriptSequenceMetadata` sounds better.

Reply:
Updated data to metadata and resolved conflicts.